airflow/providers/google/cloud/log/gcs_task_handler.py

```diff
@@ -165,7 +165,7 @@ def gcs_write(self, log, remote_log_location, append=True):
                 tmpfile.flush()
                 self.hook.upload(bkt, blob, tmpfile.name)
         except Exception as e:  # pylint: disable=broad-except
-            self.log.error('Could not write logs to %s: %s', remote_log_location, e)
+            self.log.error(f'Could not write logs to {remote_log_location}: {e}')
```

Comment:
It's not correct.
http://pylint-messages.wikidot.com/messages:w1201

Reply:
Thanks fixed.